pkg/ddc/alluxio/health_check_test.go

```diff
@@ -1,3 +1,21 @@
+/*
+
+Copyright <year> The <project-name> Author.
```

Comment:
This should be `Copyright 2021 The Fluid Author.`

Reply:
I have revised it. Please check it.